application/fpga/python/example-driver.py

```diff
@@ -61,9 +61,9 @@
 	ret = (0, 0)
 	while not ret[1]:
 		ret = driver.Steps(steps1, steps2, steps3, 1, 0, 1, gripper, toolRotation)
-	if not ret[1]:
-		errors += 1
-		print('Error', errors)
+		if not ret[1]:
```

Comment:
I don't think this code makes any sense. It wasn't counting anything before and now it is counting commands that were not queued as errors. Counting either not queued commands or errors in a loop of 20 iterations doesn't bring any value. I think it a leftover from investigation on how accurately sdk keeps track of steps, so I would just remove that piece to not confuse other users with it. Curious minds would try to understand the point of that code and would just waste time. You did get it right that those not queued commands, but that counter will always be 0 because the default buffer size in firmware is 200, so all 20 commands will be queued.